src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -11,21 +11,15 @@ const MAX_REPOS_RETURNED = 20;
 export const GithubCreateBranchGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const {
 		githubToken,
-		gitHubAppConfig
+		gitHubAppConfig,
+		jiraHost
 	} = res.locals;
-	const jiraHost = req.query?.jiraHost as string;
 
 	if (!githubToken) {
 		req.log.warn(Errors.MISSING_GITHUB_TOKEN);
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
-	if (!jiraHost) {
```

Comment:
This is being handled in the new temp middleware function, so just removing it since its duplicated

Reply:
/nit
But it doesn't harm to have it here, as it can be #fail-fast ? If somehow jiraHost is null and it slip through to bellow `await Subscription.getAllForHost(jiraHost....`, we want it to fail earlier?

Reply:
mmmm ill bring it back in, it will check jirahost from locals not query but you are right its still worth checking.